src/four_gray.toit

```diff
@@ -139,6 +139,10 @@ class OpaquePixmapTexture extends BitmapTextureBase_:
 // an arbitrary color.
 class PbmTexture extends PbmTexture_:
   // The byte array passed in must be a valid binary-mode (P4) PBM file.
+  // If $bytes is a literal then it will be used directly from flash unless the pixel
+  //   drawing methods on this are used, in which case the underlying byte
```

Comment:
Maybe?
```suggestion
  // If $bytes was created as a constant literal then it will be used directly from flash. However,
  //   if the pixel drawing methods on this are used, then the underlying byte
```

Also: shouldn't these be toitdocs?